### DISCUSSION TITLE
🙋 Q&A

### DISCUSSION THREAD

**Discussion by Eis4TY** [Eis4TY/XR-Stereoscopic-Viewer#10] [Q&A]
- **Ask Anything**: Got a question? Post it here! No query is too small or too big.
- **Share Knowledge**: Know the answer to someone's question? Don't hesitate to share your insights!
- **Upvote Answers**: If you find a response helpful, give it a thumbs up! The most helpful answers will rise to the top.

Before posting, please take a look at our guidelines to ensure our community remains helpful and positive:

1. Search the existing questions to avoid duplicates.
2. Keep your questions clear and concise.
3. Be respectful and constructive in your communication.
4. Remember to mark your question as answered when you've received a satisfactory response.

This Q&A is a community effort, and every question matters. We can't wait to see the amazing support and collaboration happening here. Let's get started!

Happy Discussing!

**Comment by Naiselim** [comment]
我是一位Unreal Engine开发者，对Unity的了解并不完全，我想学习一下如何在UE上实现这一套随动渲染以及周边模糊的效果。但是阅读分析源码只看到了GaussianBlur_R和GaussianBlur_L等Shader文件，并不知道是怎么在Unity里使用的，不知道是否方便指点一下如何进行近一步学习了解它底层原理呢？

**Reply by Eis4TY** [reply]
这不是真的实时模糊。为了节省性能，只是在窗口上显示另一张已经高斯模糊的背景图

**Comment by burritosoftware** [comment]
Could a build for SteamVR be created so we can view videos on a PC?

**Reply by Eis4TY** [reply]
That sounds good, but I'm sorry, I'm too busy on my daily work to expand this project to other platforms.

**Comment by mengran123** [comment]
Hello developer, thank you for developing this software. Could you please add a fast forward and rewind function for video playback? thanks

**Comment by Eis4TY** [comment]
Thanks for you advices, I will think about it.

**Comment by code-savior** [comment]
有没有可能直接放在AVP中作为窗口应用执行呢

**Reply by Eis4TY** [reply]
我感觉不行，但你可以试试😄

**Comment by huangyuan2020** [comment]
您好，我想了解下IPD是啥，以及在代码中具体是怎么通过IPD是调整图片的，请问这个是直接看代码，还是有什么别的资料推荐吗？

**Reply by Eis4TY** [reply]
IPD 是瞳距调节功能。Inter-Pupillary Distance 的简称。会影响立体视觉的深度和尺寸感知，你可以查一下瞳距对立体视觉的影响。B 站推荐这个系列的视频，讲的比较系统。
https://space.bilibili.com/40043075/favlist?fid=2259950075&ftype=create

**Comment by jianhuaxiao** [comment]
你好，这个支持sbs格式的视频， vision pro拍摄的MV-HEVC格式不支持吗？

**Reply by Eis4TY** [reply]
不支持，因为这个是要硬件编解码的。
